src/modules/public/blocklistAccess/selectors.ts

```diff
@@ -0,0 +1,11 @@
+import { RootState } from '../../../modules';
+import { BlocklistAccessState } from './reducer';
+
+export const selectBlacklistAccessLoading = (state: RootState): BlocklistAccessState['loading'] =>
```

Comment:
```suggestion
export const selectBlocklistAccessLoading = (state: RootState): BlocklistAccessState['loading'] =>
```
same for other selectors